src/repairCrew.cpp

```diff
@@ -179,11 +179,13 @@ void RepairCrew::update(float delta)
             position = sf::Vector2f(pos);
 
             ESystem system = ship->ship_template->getSystemAtRoom(pos);
-            if (system != SYS_None)
+            if (system != SYS_None && ship->systems[system].health < ship->systems[system].health_max)
```

Comment:
This check is flawed, as it won't unhack systems that are above max health now.

Reply:
Do you think I need to put a specific part of code like this ?
`ship->systems[system].health = std::min(ship->systems[system].health_max , ship->systems[system].health)`